lightning/src/offers/offer.rs

```diff
@@ -208,7 +209,7 @@ impl MetadataStrategy for DerivedMetadata {}
 macro_rules! offer_explicit_metadata_builder_methods { (
 	$self: ident, $self_type: ty, $return_type: ty, $return_value: expr
 ) => {
-	/// Creates a new builder for an offer setting the [`Offer::description`] and using the
+	/// Creates a new builder for an offer setting an empty [`Offer::description`] and using the
 	/// [`Offer::signing_pubkey`] for signing invoices. The associated secret key must be remembered
```

Comment:
IMO not necessary to call out the description:
```suggestion
	/// Creates a new builder for an offer using the
	/// [`Offer::signing_pubkey`] for signing invoices. The associated secret key must be remembered
```